html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input name="email" id="email">
+        </div>
+      </div>
+      <div>
+        <label for="whycode">Why is coding cool?</label>
+        <div>
+          <select name="whycode">
+            <option>Select and option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </div>
+      </div>
+      <div>
+        <p><label for="code">Favorite Language (choose one)</label></p>
+        <div>
+          <input type="radio" name="code" value="HTML">HTML
```

Comment:
each of these input needs its own label, you may wrap the inputs with the labels if desired